### DISCUSSION TITLE
Live

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#332] [Q&A] [answered]
Alguien ya entro al live no se ve en mi equipo

**Comment by Lfer1111** [comment]
a mi tampoco me deja, que pasara?

**Comment by AngelAlfaro92** [comment, marked as answer]
Al parecer aun no ha iniciado tendriamos que esperar

**Comment by gersondlarosa** [comment]
Ok muchísimas gracias

**Comment by Lfer1111** [comment]
ya posteo el comander rodrigo que se quedo sin luz, hasta mañana el live

**Comment by AngelAlfaro92** [comment]
Actualizacion, ya me fijé en teams, el MC publicó que hoy no habrá clase porque se le fue la luz

**Comment by Ingrzs** [comment]
Hoy no habrá live, el Mc no cuenta con luz.
Mañana y viernes ala misma hora

**Comment by Lfer1111** [comment]
gracias banda, seguimos pendientes

**Comment by gersondlarosa** [comment]
Muchas gracias a todos
